Content.Server/DeltaV/Station/Systems/CaptainStateSystem.cs

```diff
@@ -0,0 +1,87 @@
+using Content.Server.Chat.Systems;
+using Content.Server.DeltaV.Station.Components;
+using Content.Server.GameTicking;
+using Content.Server.Station.Systems;
+using Content.Shared.Access.Systems;
+
+namespace Content.Server.DeltaV.Station.Systems;
+
+public sealed class CaptainStateSystem : EntitySystem
+{
+    [Dependency] private readonly ChatSystem _chat = default!;
+    [Dependency] private readonly StationSystem _stationSystem = default!;
+    [Dependency] private readonly GameTicker _gameTicker = default!;
+    [Dependency] private readonly IEntityManager _entityManager = default!;
```

Comment:
naming for dependencies should be 1 word only, `_station`, `_ticker`, `_entity`
also sort alphabetically